raft.go

```diff
@@ -937,8 +1110,8 @@ func (r *Raft) leaderLoop() {
 				v.respond(nil)
 			}
 
-		case p := <-r.peerCh:
-			p.respond(ErrLeader)
+		case future := <-r.configurationChangeChIfStable():
```

Comment:
Looking through the code I agree this looks fine. We should be getting the commit channel advancing us as the config change is accepted by the peers and that will cause us to loop again. These items won't change outside leader loop.